### PR TITLE
Add GC.logging_enabled()

### PR DESCRIPTION
**Pull request by fingolfin**
... so one can query the state of GC.enable_logging().

### CODE REVIEW

**Review by vchuravy — APPROVED** [review]

File: base/gcutils.jl [hunk 1]
```diff
@@ -259,4 +259,13 @@ function enable_logging(on::Bool=true)
     ccall(:jl_enable_gc_logging, Cvoid, (Cint,), on)
 end
 
+"""
+    GC.is_logging()
+
+When turned on, print statistics about each GC to stderr.
```
Comment (ericphanson) [inline base/gcutils.jl:265]: This sounds like the docstring for enable logging, not querying that state

File: base/gcutils.jl [hunk 2]
```diff
@@ -259,4 +259,13 @@ function enable_logging(on::Bool=true)
     ccall(:jl_enable_gc_logging, Cvoid, (Cint,), on)
 end
 
+"""
+    GC.is_logging()
```
Comment (ararslan) [inline base/gcutils.jl:263]: The name makes it sound like it's querying whether logging is currently happening but it's actually querying whether logging is enabled. Perhaps `GC.logging_enabled()` would be a more accurate name? It matches the C global being accessed, plus it mirrors other functions that query whether something is enabled, e.g. `inlining_enabled` and `coverage_enabled` in `Core.Compiler`.

File: base/gcutils.jl (hunk 1 above)
Comment (fingolfin) [reply base/gcutils.jl]: oops, fixed now

File: base/gcutils.jl (hunk 2 above)
Comment (fingolfin) [reply base/gcutils.jl]: fine by me. changed

**Review by ararslan — APPROVED** [review]

**Comment by ararslan** [comment]
A NEWS entry for this would be nice as well.

**Comment by fingolfin** [comment]
Added a NEWS.md item (pity I didn't see this comment earlier and so missed 1.10 :-/)

**Pull request merged**